test/unit/errors-tests.js

```diff
@@ -0,0 +1,166 @@
+"use strict";
+const { assert } = require("chai");
+const { napiErrorHandler } = require("../../lib/new-utils");
+const { throwTestError } = require("../../index");
+const {
+    ArgumentError,
+    AuthenticationError,
+    BusyConnectionError,
+    DriverError,
+    DriverInternalError,
+    NoHostAvailableError,
+    NotSupportedError,
+    OperationTimedOutError,
+    ResponseError,
+} = require("../../lib/errors");
+
+class SomeJsClass {
+    throwError = napiErrorHandler(function (errorType) {
+        return throwTestError(errorType);
+    });
+    throwMessage = napiErrorHandler(function (message) {
+        return throwTestError("Error", message);
+    });
+    throwSimple = napiErrorHandler(function () {
+        throw new Error("Simple error");
+    });
+}
```

Comment:
:question: What's the purpose of each of these?

Reply:
I added comments to explain each function. They are here to test if the wrapper handles correctly different types of error, errors with a message and errors thrown from JS instead of napi.